src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -217,7 +217,10 @@ export class SamCliLocalInvokeInvocation {
 
         await this.args.invoker.invoke({
             options: {
-                env: this.args.environmentVariables,
+                env: {
+                    ...process.env,
```

Comment:
Is the change in `src/shared/sam/localLambdaRunner.ts` the necessary fix? Curious how it is necessary to provide `process.env` now if this was not the case before.

Reply:
the `localLambdaRunner.ts` change is not necessary, but it is still a similar kind of bug.